Library/Homebrew/formula.rb

```diff
@@ -2556,7 +2556,7 @@ def install; end
   # @api public
   sig {
     params(
-      paths:        T.any(T::Array[T.untyped], String, Pathname),
+      paths:        T.any(T::Enumerable[T.untyped], String, Pathname),
```

Comment:
`paths` can only be `String`-like or `Pathname`-like, so I wonder if we can/should avoid `T.untyped` here. Maybe we can just fix the callers if needed?